tools/dependencies.props

```diff
@@ -1,20 +1,4 @@
 <Project>
```

Comment:
Make sure to remove this file before merging.

Reply:
Yup yup. Removed the file now. I was just testing locally if the build will fail when there are references to this file and I remove it. I removed the references to the file now too and everything should succeed.